gsm.yaml

```diff
@@ -0,0 +1,74 @@
+package:
+  name: gsm
+  version: 1.0.22
+  epoch: 0
+  description: Shared libraries for GSM speech compressor
+  copyright:
+    - license: TU-Berlin-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - automake
+      - autoconf
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha512: f4284f096ed4d606ddece5c1b28edb63feacc26a0b664f1350c040dbf50113d27bedfd9896bf1620cb4fc3a6c8f6a5244955a6c625ded833dcee607c63c97dd2
+      uri: https://www.quut.com/gsm/gsm-${{package.version}}.tar.gz
+      strip-components: 1
+
+  - uses: patch
+    with:
+      patches: gsm-64bit.patch
+
+  - uses: patch
+    with:
+      patches: gsm-makefile.patch
+
+  - runs: |
+      export RPM_OPT_FLAGS="$CFLAGS -fPIC"
+      make all
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/usr/bin \
+      	"${{targets.destdir}}"/usr/include/gsm \
+      	"${{targets.destdir}}"/usr/lib \
+      	"${{targets.destdir}}"/usr/share/man/man1 \
+      	"${{targets.destdir}}"/usr/share/man/man3
+
+      make install \
+      	INSTALL_ROOT="${{targets.destdir}}"/usr \
+      	GSM_INSTALL_INC="${{targets.destdir}}"/usr/include/gsm \
+      	GSM_INSTALL_LIB="${{targets.destdir}}"/usr/lib
+
+      ls -latr ./lib
+
+      cp -p lib/libgsm.so.1.0.12 "${{targets.destdir}}"/usr/lib/
+      ln -s libgsm.so.1.0.12 "${{targets.destdir}}"/usr/lib/libgsm.so.1
+      ln -s libgsm.so.1.0.12 "${{targets.destdir}}"/usr/lib/libgsm.so
+
+      # the makefile is really broken
+      install -Dm644 inc/gsm.h \
+      	-t "${{targets.destdir}}"/usr/include/gsm/
+      ln -s gsm/gsm.h "${{targets.destdir}}"/usr/include/gsm.h
+      rm -f "${{targets.destdir}}"/usr/lib/lib*.a
+
+  - uses: strip
+
+subpackages:
+  - name: "gsm-tools"
+    description: "GSM speech compressor tools"
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/usr
+          mv "${{targets.destdir}}"/usr/bin/* "${{targets.subpkgdir}}"/usr/
```

Comment:
This doesn't look right. Won't this put everything in `/usr/bin` into the subpackage's `/usr/`?

Reply:
Yeah, it should be

```suggestion
          mv "${{targets.destdir}}"/usr/bin "${{targets.subpkgdir}}"/usr/
```